### ISSUE TITLE
关于面部风格化功能

### ISSUE THREAD

**Issue by RainEmy** [InstantStyle/InstantStyle#31]
感谢作者开源这么牛逼的项目，请问近期会发布面部风格化功能吗，我已经迫不及待想测试了！

**Comment by haofanwang** [comment]
Yes, we are waiting for the final review of this [PR](https://github.com/huggingface/diffusers/pull/7668). Will support with InstantID soon.
